wfe/context.go

```diff
@@ -13,19 +13,17 @@ import (
 )
 
 type requestEvent struct {
-	ID            string `json:",omitempty"`
-	RealIP        string `json:",omitempty"`
-	ClientAddr    string `json:",omitempty"`
-	Endpoint      string `json:",omitempty"`
-	Method        string `json:",omitempty"`
-	Errors        []string
-	Requester     int64                  `json:",omitempty"`
-	Contacts      *[]string              `json:",omitempty"`
-	RequestNonce  string                 `json:",omitempty"`
```

Comment:
Good to know! I'll put them back for now. FYI we also log response nonces in the Nginx logs. Maybe we should remove it there and just use it in WFE?